clang/include/clang/Sema/Sema.h

```diff
@@ -10615,6 +10615,8 @@ class Sema final {
     ChangedStateAtExit
   };
 
+  void DiagnoseMissingFormatAttributes(NamedDecl *FDecl, SourceLocation Loc);
```

Comment:
```suggestion
  void DiagnoseMissingFormatAttributes(const NamedDecl *FDecl, SourceLocation Loc);
```
(If we can't add the `const` here because it's viral for some reason, then I don't insist on this change.)